.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = tab
+indent_size = 4
+insert_final_newline = true
+end_of_line = lf
+
+[*.py]
+indent_style = space
+max_line_length = 99
```

Comment:
120

Reply:
I was referring to the [official PEP8 guide, paragraph 5](https://peps.python.org/pep-0008/#maximum-line-length).
But if you opt for 120, there's no problem. Should I change it to 120?

Reply:
Makes sense, I have a 2k screen, so 99 feels small but yes I agree with PEP8, keep 99, thanks!